packages/flow-designer/src/components/ZoomHandler.component.tsx

```diff
@@ -35,25 +35,21 @@ class ZoomHandler extends React.Component<Props, State> {
 			.scaleExtent([1 / 4, 2])
 			.on('zoom', this.onZoom)
 			.on('end', this.onZoomEnd);
-	}
 
-	UNSAFE_componentWillMount() {
```

Comment:
This init the state at mount, move that in constructor.
We need to keep the class for extend